.github/workflows/docker.yml

```diff
@@ -17,10 +17,10 @@ on:
   workflow_dispatch:
     inputs:
       repo_ref:
-        required: true
+        required: false
```

Comment:
An example building log is here:
https://github.com/RunningLeon/lmdeploy/actions/runs/7334739555